platform/ios/MGLMapView_IBAdditions.m

```diff
@@ -0,0 +1,57 @@
+#import "MGLMapView_IBAdditions.h"
+
+#import <objc/runtime.h>
+
+#import "NSProcessInfo+MGLAdditions.h"
+
+@implementation MGLMapView (IBAdditions)
+
++ (NSSet *)keyPathsForValuesAffectingLatitude
+{
+    return [NSSet setWithObject:@"centerCoordinate"];
+}
+
+- (double)latitude
+{
+    return self.centerCoordinate.latitude;
+}
+
+- (void)setLatitude:(double)latitude
+{
+    NSNumber *pendingLongitude;
+    if ((pendingLongitude = objc_getAssociatedObject(self, @selector(longitude))))
+    {
+        objc_setAssociatedObject(self, @selector(latitude), nil, OBJC_ASSOCIATION_RETAIN_NONATOMIC);
```

Comment:
I know this associated object stuff is crazy, but I wanted two separate numeric inspectables instead of one comma-separated string inspectable (since it wouldn’t be clear whether latitude or longitude comes first). Setting only longitude or latitude has no effect: the coordinate remains (NaN, NaN).